protocol-spec.tex

```diff
@@ -47,22 +47,29 @@ \section{Modifications to Batman-Adv protocol spec}
 
 				They then forward the ack on along the route. Other nodes along the route can use the ack message to make the same estimate. They compare the traffic they have forwarded from the source to the traffic that the destination says it received from the source.
 
-				This way, all the nodes on the route can make an estimate about the true quality of the route.
+				This way, all the nodes participant in a connection for at least one full ack period can make an estimate about the true quality of the route.
 
 				This provides enough information for the network at large to take action against false routing metrics. Each node participating in an active connection will be able to estimate an accurate quality metric to at least one of the ends of the connection, once every ack period.
 
-				At the very least, this lets nodes build up accurate information on routes they are using. If a route's true quality is less than other available routes to that destination, another route will be switched to within one ack period.
-
-				Blacklisting could also be an option. If a neighbor is consistently advertising innacurate quality metrics, something is wrong. There could be a wide range of blacklisting policies, and a lot of details involved in tweaking them.
-
-				Another technique could be to allow nodes to build up trust with their neighbors, and select routes from trusted neighbors more often. Unknown nodes would not recieve much traffic until they demonstrated the accuracy of their routes. This might act to mitigate "sybil" attacks where nodes quickly assume new identities.
+				This method suffers from being a passive throughput metric rather than an active one. Passive metrics may not fully show the throughput of a connection, but active metrics require additionally protocol overhead and run into the challenge of hiding the fact that they are a throughput test to prevent malicious nodes from prioritizing them and giving an inaccurate picture of the routes true throughput. 
+				
+				Since we can not confirm if a node is lying or simply has a flawed method to estimate it's own link bandwidth the ack metric should be used to adjust the advertised throughput of a given route not for blacklisting or forms of the network attempting to pushing malicious nodes. Any node that sees an applicable ack will adjust the advertised throughput for that route in it's own originator messages meaning large swaths of the network will begin to participate in throughput correction in a single ack period and most nodes will be able to participate in several simultaneous throughput corrections without additional overhead beyond the packet counters required for the first correction.
```

Comment:
> Since we can not confirm if a node is lying or simply has a flawed method to estimate it's own link bandwidth the ack metric should be used to adjust the advertised throughput of a given route not for blacklisting or forms of the network attempting to pushing malicious nodes.

Regardless of what the node's intentions are, their inaccuracy needs to be adjusted for.